src/NUnitEngine/nunit.engine.core/RuntimeFramework.cs

```diff
@@ -201,6 +201,8 @@ private Version GetClrVersionForFramework(Version frameworkVersion)
                             return new Version(5, 0, 1);
                         case 6:
                             return new Version(6, 0, 0);
+                        case 7:
```

Comment:
@stevenaw Yes, but not worth the effort right here as this method doesn't exist in the 4.0 builds. Once released, 3.15.1 will become my new "final" 3.x release - unless another critical bug shows up!

Reply:
Thanks @CharliePoole I completely agree and understand about "not right now" 🙂

I'm mostly asking because I've seen a few places where we have this hard-coded version mapping and I wasn't sure if there'd been discussion about it yet, or considered and ruled out for some reason. I didn't realize it had already been adjusted in the 4.0 build.